iceoryx_hoofs/test/moduletests/test_resizeable_lockfree_queue.cpp

```diff
@@ -29,14 +29,16 @@ using namespace ::testing;
 // use a non-POD type for testing (just a boxed version of int)
 struct Integer
 {
+    // NOLINTNEXTLINE(hicpp-explicit-conversions) required for typed tests
     Integer(uint64_t value = 0)
         : value(value)
     {
     }
 
-    uint64_t value{0};
+    uint64_t value = 0;
```

Comment:
Out of curiosity, did clang-tidy complain about this or is it just personal preference?

Reply:
I do not remember. Will check later. Personally I can live with either syntax for primitive types but prefer `=` as it is closer to other languages.

Reply:
@elBoberido It is not required and hence undone. I believe clang-tidy complained when my language server was not working correctly for some reason but cannot reproduce it.

In my opinion both are fine for primitive types.